templates/Pages/about.php

```diff
@@ -41,6 +41,25 @@
         <li><strong>IO500 List</strong>: the subset from the full list with only the best submission per storage system</li>
         <li><strong>10 Node Challenge List</strong>: the subset from the full list with only the best submission per storage system ran on exactly ten nodes</li>
     </ul>
+    
+    <p>
+        Please note that the Ranked Lists will effectively <i>hide</i> some submissions due to the fact that they only show the best submission per storage systems.  Therefore, 
+        if a storage system has multiple submissions, only the top one by overall score will be visible in the Ranked Lists but all submissions will appear in the other lists.  
```

Comment:
and as a result of the above change "... only one score will be visible in the Ranked Lists, but ..."